React/Base/RCTBridgeModule.h

```diff
@@ -441,7 +441,7 @@ typedef NSURL * (^RCTBridgelessBundleURLGetter)(void);
                            andSetter:(RCTBridgelessBundleURLSetter)setter
                     andDefaultGetter:(RCTBridgelessBundleURLGetter)defaultGetter;
 - (void)resetBundleURL;
-@property NSURL *bundleURL;
+@property (strong) NSURL *bundleURL; // TODO(macOS GH#774)
```

Comment:
While here I figure I'll ask - is there an actual accessor for this? Seeing it's not marked as strong makes me also curious if we should be `nonatomic`

Reply:
Discussed offline, and we decided it should be fine as is.